### PR TITLE
[jsonrpc] CEpgInfoTag: fix serialization of "runtime" as an integer (in minutes) instead of as a string

### PR DESCRIPTION
**Pull request by Montellese**
This fixes an inconsistency between the JSON-RPC API definition for the `runtime` property of a PVR broadcast and the actual value being returned.

### CODE REVIEW

**Comment by Razzeee** [comment]
should be fine

File: xbmc/interfaces/json-rpc/schema/version.txt [hunk 1]
```diff
@@ -1 +1 @@
-6.33.0
+6.33.1
```
Comment (MilhouseVH) [inline xbmc/interfaces/json-rpc/schema/version.txt]: Needs to be bumped `6.33.1` to `6.33.2`.

Comment (Montellese) [inline xbmc/interfaces/json-rpc/schema/version.txt]: Why? Current master is at 6.33.0. I know that there are other PRs that touch this version but the order of merge will decide which PR will have to bump to which version.

Comment (MilhouseVH) [inline xbmc/interfaces/json-rpc/schema/version.txt]: Current master is 6.33.1, bumped by #8654.

https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/json-rpc/schema/version.txt

**Comment by Montellese** [comment]
jenkins build and merge

**Pull request merged**